chromium/background.js

```diff
@@ -710,6 +712,7 @@ Object.assign(exports, {
   all_rules,
   initializeStoredGlobals,
   urlBlacklist,
+  state,
```

Comment:
Adding this `state` stuff can be removed. I was using it to check the value of js variables after interacting with UI. But instead, now I just verify the UI is in the correct state after refreshing.